### PR TITLE
Monkeys drop forced two handed items

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

Monkeys cannot wield twohanded items.
But are not forced to drop items that must be wielded two handed.
This forces monkeys to drop items if they need to be two handed.

## Why It's Good For The Game

Fixes monkeys...

## Changelog
:cl:
fix: Monkeys are no longer able to hold items that require two hands.
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
they will still try to pick them up with the AI which would break now, you'd need to fix that as well.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
What Floyd said, make monkey AIs not want to pick up two-handed items.

**Comment by nightred** [comment]
Added the trait for items needing two hands to carry, and a check in monkey AI for that.

File: code/datums/ai/monkey/monkey_behaviors.dm [hunk 1]
```diff
@@ -25,6 +25,10 @@
 		finish_action(controller, FALSE)
 		return
 
+	if(HAS_TRAIT(target, TRAIT_NEEDS_TWO_HANDS)) //Can't carry items that need two hands as a monkey
```
Comment (Qustinnus) [inline code/datums/ai/monkey/monkey_behaviors.dm:28]: This should be done in the ai controller so it doesn't bother going for weapons it cant go for.

Comment (nightred) [reply code/datums/ai/monkey/monkey_behaviors.dm]: I have moved this to the controller side.

**Review by ghost — APPROVED** [review]
if you tested it still works this all looks good 2 me

**Comment by nightred** [comment]
> if you tested it still works this all looks good 2 me

I tested local, and monkeys picked up items but did not pick up potted plants.

**Pull request merged**